routers/web/repo/issue.go

```diff
@@ -135,6 +135,7 @@ func MustAllowPulls(ctx *context.Context) {
 }
 
 func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption util.OptionalBool) {
+	AddHeaderCtxData(ctx)
```

Comment:
I guess we shouldn't call it here. It's better to call such "template data" related function at every route handler's entry.

Otherwise it's unclear when and how such function is called/affects.